src/tools/opt-dist/src/tests.rs

```diff
@@ -59,26 +59,19 @@ pub fn run_tests(env: &Environment) -> anyhow::Result<()> {
         .join(format!("llvm-config{}", executable_extension()));
     assert!(llvm_config.is_file());
 
-    let config_content = format!(
-        r#"profile = "user"
-change-id = 115898
+    let profile = "profile=user";
```

Comment:
The profile might not even be needed, if we're reusing an existing `config.toml`. It was only there because the config file was created from scratch and some profile had to be there.

Reply:
> some profile had to be there

profiles are optional IIRC, I don't use one.